src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] ball = new Ball[3];
```

Comment:
You are not using an array. You can delete it.
To get the required number of balls, just specify 3 instead of ball.length